tutorials/Example_Track_on_Radar_Segment_on_Satellite/track_on_radar_segment_on_satellite.ipynb

```diff
@@ -0,0 +1,1245 @@
+      "text/plain": [
+       "<Figure size 800x800 with 3 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# In Py-ART's graphing suite, there is a display class similar to RadarMapDisplay,\n",
+    "# but for grids. To plot the grid:\n",
+    "fig = plt.figure(figsize=[8, 8])\n",
+    "plt.axis('off')\n",
+    "\n",
+    "display = pyart.graph.GridMapDisplay(grid)\n",
+    "display.plot_grid('reflectivity', level=3, vmin=0, vmax=60, fig=fig)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xr_grid = grid.to_xarray()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "numpy.datetime64('2021-05-26T15:56:15.477525888')"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "goes_data['time_bounds'][0].values"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "numpy.datetime64('2021-05-26T15:56:15.477525888')"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "goes_data['time_bounds'][0].values"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/var/folders/bj/m6g82c6n41g83y3_dx02y7ch0000gp/T/ipykernel_29580/584919884.py:35: RuntimeWarning: invalid value encountered in sqrt\n",
+      "  r_s = (-1.0 * b_var - np.sqrt((b_var**2) - (4.0 * a_var * c_var))) / (2.0 * a_var)\n"
+     ]
+    }
+   ],
+   "source": [
+    "llons, llats  = lat_lon_reproj(goes_data)\n",
+    "full_goes_data = goes_data['CMI_C10'].values\n",
+    "in_goes_for_tobac = xr.Dataset(data_vars=\n",
```

Comment:
This variable name is also slightly obscure, maybe something like `goes_brightness_temperature` would be clearer?